moment-timezone.js

```diff
@@ -330,6 +350,16 @@
 		}
 	}
 
+	function findYearLike(year) {
+		for (var j = 2038; j > 2000; j--) {
```

Comment:
If we aren't worried about strange exceptions (like when a timezone skips a day entirely), we use the fact that there's a 28-year cycle in which the calendar repeats itself, instead of performing a loop.

Reply:
Yes, I already fixed that, will push tomorrow, it was beta version to check if that logics (of searching similar years) will work at all